common/jenkins-slaves/python/Dockerfile.rhel7

```diff
@@ -30,7 +31,7 @@ RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" \
 # Upgrade PIP
 RUN pip3 install --upgrade pip \
     && pip3 -V \
-    && pip3 install virtualenv pycodestyle
+    && pip3 install virtualenv==${VENV_VERSION}
```

Comment:
Are there reasons this is not using Pythons standard library venv module (https://docs.python.org/3/library/venv.html)?

Reply:
I have checked if it is better to use venv instead of virutalenv and here you can find strong reasons to keep using virutalenv --> https://virtualenv.pypa.io/en/latest/

Reply:
I would still vote to use the builtin module unless we actually run into limitations.

Reply:
well, we want to enable users with other python versions for example, isn't that enough reason?

Reply:
I see a point on being leaner and also that there is still the chance that if a user needs virutalenv he/she can install it via venv hehe, ok!!! but what abouth offering both venv and virutalenv from the beginning? :)

Reply:
I am not convinced, but I also did not see a problem.